e2e/tests/test_cases/single_replica_node_down.robot

```diff
@@ -0,0 +1,118 @@
+*** Settings ***
+Documentation    Single replica node down
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-down/single-replica-node-down/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/recurringjob.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Library    OperatingSystem
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+${SHUTDOWN_WAIT_DURATION}    180
+
+*** Test Cases ***
+Single Replica Node Down Deletion Policy do-nothing With RWO Volume Data Locality disabled
+    When Create persistentvolumeclaim 0 using RWO volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Write 100 MB data to file data in deployment 0
+    And Update persistentvolumeclaim 0 volume replica count to ${1}
+    And Delete replica of deployment 0 volume on replica node
+    And Delete replica of deployment 0 volume on volume node
+    And Power off volume node of deployment 0
+    And Sleep   ${SHUTDOWN_WAIT_DURATION}
+    Then Wait for volume of deployment 0 attaching
```

Comment:
There is a step `The workload pod will get stuck in the terminating state.` in the original manual test case. Does it need to be verified as well?

Reply:
Updated
```
    And Wait for deployment 0 pods on volume node kept in state Terminating
    And Wait for deployment 0 pods on replica node kept in state ContainerCreating
```

Reply:
Could you help elaborate more on why do we need 2 separate steps here?

Reply:
Why node info is required when waiting for a pod state?

Reply:
Updated keyword name below to more close to the test plan. (change `kept` to `stuck`)
```robotframework
Wait for ${workload_kind} ${workload_id} pods on ${replica_locality} stuck in state ${expect_state}
```

Because in some scenarios there will not be only 1 pod in the test workload(1 stuck on volume node, 1 stuck in creating etc.).  and then it need node info to check pod state on specific node for the keyword

Back to the review comment
Step 
`The workload pod will get stuck in the terminating state.` 
should match 
`And Wait for deployment 0 pods on volume node stuck in state Terminating`

Step 
`The replacement workload pod will get stuck in the ContainerCreating state.` 
should match 
`And Wait for deployment 0 pods on replica node kept in state ContainerCreating`

Reply:
In the original test case, it seems we don't need to know the pod running on which node? It only knows the original pod and replaced pod. Can we achieve this by retrieving pod's `creationTimestamp` or `startTime`?

Reply:
IIRC, the origin pod will be volume/engine node, after the volume node powed off, the replaced pod should create on the other scheduleable node. 

The step `The replacement workload pod will get stuck in the ContainerCreating state.` be performed by `And Wait for deployment 0 pod stuck in ContainerCreating on another node`, thank you. Or if it's necessary to use `startTime`, I will update it, thank you.